packages/angular/common/src/providers/nav-controller.ts

```diff
@@ -37,9 +37,9 @@ export class NavController {
     if (router) {
       router.events.subscribe((ev) => {
         if (ev instanceof NavigationStart) {
+          // restoredState is set if the browser back/forward button is used
           const id = ev.restoredState ? ev.restoredState.navigationId : ev.id;
-          this.guessDirection = id < this.lastNavId ? 'back' : 'forward';
-          this.guessAnimation = !ev.restoredState ? this.guessDirection : undefined;
+          this.guessDirection = this.guessAnimation = id < this.lastNavId ? 'back' : 'forward';
```

Comment:
One bug I noticed is animations now play when using the back/forward buttons in the `sidemenu` app (clicking items in the menu don't play an animation). I'll see if there's a solution for this.

Reply:
Uploading Screen Recording 2023-10-13 at 12.01.33 PM.mov…

Reply:
Any news regarding this, @liamdebeasi ?
If it help, I can also take another look at the bug :)

Reply:
@liamdebeasi I had a look at the issue. In order to resolve it I needed some of the changes from my other open PR #28297.
I applied the fix to both branches and adapted the test as we don't need the 500ms wait in the `root` case anymore now. Hope that helps!

Reply:
@liamdebeasi Any chance you have some time to take a look at the fix? 😇

Reply:
I have not. The ticket has been pulled into our current sprint which just started, so I should be able to look at it soon.